File: src/main/kotlin/net/wavem/uvc/rms/common/domain/job/result/JobResult.kt
package net.wavem.uvc.rms.common.domain.job.result

data class JobResult(
    val status: String?,
    val startTime: String?,
    val endTime: String?,
    val startBatteryLevel: Int?,
    val endBatteryLevel: Int?,
    val dist: Int?
) {
}

File: src/rms/response/event/domain/taskInfo/job_result.py

class JobResult():
    def __init__(self, status=None, startTime=None, endTime=None, startBatteryLevel=None, endBatteryLevel=None, dist=None) -> None:
        self.status = status
        self.startTime = startTime
        self.startBatteryLevel = startBatteryLevel
        self.endBatteryLevel = endBatteryLevel
        self.dist = dist